overview.md

```diff
@@ -40,36 +38,36 @@ SubDAOs are expected to maintain a synergistic relationship with Maker Core thro
 
 Moving much of the developmental, operational, and governance work to SubDAOs makes Maker Core simpler and more resilient.
 
-### [Scopes and Scope Frameworks](maker-core/scopes-and-frameworks.md)
+### [Scopes and Scope Artifacts](maker-core/scopes-and-artifacts.md)
 
-Maker Core will consist of twelve Scopes. A Scope is a broad focus area for any work done to operate or improve the protocol. 
+Maker Core will consist of [five Scopes](maker-core/list-of-scopes.md). A Scope is a broad focus area for any work done to operate or improve the protocol. 
 
 The goal of introducing Scopes is to limit scope creep and ensure that any work done aligns with Maker Core's goals.
 
-Scope Frameworks define the rules that SubDAOs must adhere to. Scope Frameworks will be refined over time to maximize profit and minimize risk to Maker Core.
+Scope Artifacts define the rules that SubDAOs must adhere to. Scope Artifacts will be refined over time to maximize profit and minimize risk to Maker Core.
 
-Scope Frameworks limit the governance overhead by ensuring that Maker Core only focuses on high-level decisions. They also encourage innovation among SubDAOs without allowing them to take excessive risks that may damage Maker Core. 
+Scope Artifacts limit the governance overhead by ensuring that Maker Core only focuses on high-level decisions. They also encourage innovation among SubDAOs without allowing them to take excessive risks that may damage Maker Core. 
 
-### [Constitutional Voter Committees](maker-core/cvc.md)
+### [Aligned Voter Committees](maker-core/avc.md)
 
-Constitutional Voter Committees (CVCs) are groups of voters with aligned values and views. CVCs engage in the important high-level questions of Maker Governance. Every CVC must include a [Constitutional Conserver](maker-core/constitutional-conservers.md), an entity that has pledged to support the Constitution. 
+Aligned Voter Committees (AVCs) are groups of voters with aligned values and views. CVCs engage in the important high-level questions of Maker Governance. Every AVC must include a [Aligned Conserver](maker-core/aligned-conservers.md), an entity that has pledged to support the Atlas. 
```

Comment:
- You missed one "CVC" (second sentence)
- "Every AVC must include AN [AlignMENT Conserver](maker-core/alignment-conservers.md), an entity that has the DUTY to support the Atlas." (the pledge is gone :D )